.github/workflows/update-commit-hashes.yml

```diff
@@ -2,10 +2,10 @@ name: update-commit-hashes
 
 on:
   schedule:
-    # Every day at 12:37am
+    # Every day at 7:37am UTC = 12:27am PST
     # Choose a random time near midnight because it may be delayed if there are high loads
```

Comment:
Do you want to update the comment explaining that it wants to be near midnight *PST*